aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>Is mean estimate using the rejected zernikes? It does not seem to be that different from the clipped version. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Yes, mean estimate is using the rejected Zernikes. The outlier in this dataset was not a huge outlier. I can try to find a different dataset we already have that may have a better outlier. Or actually I can artificially add in a large outlier to illustrate, I think I'll do that.</p>

Reply:
<p>I think providing an example with a claer outlier would be helpful</p>